mito-ai/src/Extensions/AiChat/ChatMessage/MarkdownBlock.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import { IRenderMimeRegistry, MimeModel } from '@jupyterlab/rendermime';
+
+
+interface IMarkdownCodeProps {
+    markdown: string;
+    rendermime: IRenderMimeRegistry;
+}
+
+const MarkdownBlock: React.FC<IMarkdownCodeProps> = ({ markdown, rendermime }) => {
+    const [renderedContent, setRenderedContent] = useState<JSX.Element | null>(null);
+
+
+    useEffect(() => {
+        const renderMarkdown = async () => {
+            const model = new MimeModel({
+                data: { ['text/markdown']: markdown },
+            });
+
+            const renderer = rendermime.createRenderer('text/markdown');
+            await renderer.renderModel(model);
+
+            const node = renderer.node;
+            setRenderedContent(<div ref={(el) => el && el.appendChild(node)} />);
+        };
+
+        renderMarkdown();
+    }, [markdown, rendermime]);
+
+    return (
+        <div>{renderedContent || <div>No content</div>} </div>
```

Comment:
Instead of returning "No content", let's just return an empty div.

Reply:
Definitely.